server/verify_test.go

```diff
@@ -1065,3 +1117,139 @@ func TestVerifyAttestationWithSevSnp(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyAttestationWithTdx(t *testing.T) {
+
+	pcr0 := uint32(0)
+	algorithms := []struct {
+		ID         uint16
+		DigestSize uint16
+		Make       func() hash.Hash
+	}{
+		{ID: 0x04, DigestSize: 0x14, Make: crypto.SHA1.New},
```

Comment:
please use the go-tpm constants: https://github.com/google/go-tpm/blob/e9722e4450de281f46cb7f4241f59695de4ed998/tpm2/constants.go#L27-L28

Reply:
You should only need to pass in the go-tpm algo. You can convert to a `crypto.Hash` using https://github.com/google/go-tpm/blob/e9722e4450de281f46cb7f4241f59695de4ed998/tpm2/structures.go#L442